### PR TITLE
Remove determine cost branch

### PR DESCRIPTION
**Pull request by tatyree**
Redirect to `lateness/in_time#edit` instead.

### CODE REVIEW

File: app/controllers/steps/details/start_controller.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 module Steps::Details
   class StartController < Steps::DetailsStepController
     def show
-      @can_start = current_tribunal_case&.cost_task_completed?
+      @can_start = current_tribunal_case.present?
```
Comment (tatyree) [inline app/controllers/steps/details/start_controller.rb]: This determination and the one in the lateness start controller can probably go altogether.  For now, however, I wanted to get rid of the cost_task reference with a minimal amount of refactoring.  Will put a ticket in to review this as a chore later.

Comment (csutter) [inline app/controllers/steps/details/start_controller.rb]: Indeed, we can chuck the start controller entirely. But happy to do that as a separate PR.

File: config/locales/appeal.yml [hunk 2]
```diff
@@ -159,31 +77,6 @@ en:
         edit:
           heading: "challenged decision header"
           lead_text: "lead text here change me"
-  activemodel:
```
Comment (csutter) [inline config/locales/appeal.yml]: These should be changed to appeal instead of dropped – they're the error messages for the forms.

Comment (tatyree) [inline config/locales/appeal.yml]: Ah, Ok.
